damage_bar.py

```diff
@@ -0,0 +1,88 @@
+import cv2
+
+class Point2D(object):
+    """
+    simple 2d point
+    """
+    def __init__(self, x, y):
+        self.data = (x, y)
+
+    def __getitem__(self, key):
+        if key == 0 or key == 'x':
+            return self.data[0]
+        elif key == 1 or key == 'y':
+            return self.data[1]
+        else:
+            return None
+
+    def __add__(self, other):
```

Comment:
Not a critical fix. This seems a bit out of the context of this object. Ideally it'll be nice to have a factory or a decorator which takes in the Point2D object and increments it, rather than one object creating a bunch of other objects